### PR TITLE
[Fix](multi-catalog) Invalidate cache when enable auto refresh catalog.

### PR DESCRIPTION
**Pull request by dutyu**
## Proposed changes

The default value of `RefreshCatalogStmt.invalidCache` is false now, but the `RefreshManager.RefreshTask` does not invoke `RefreshCatalogStmt.analyze()` so it will not invalidate the cache. This pr mainly fix this problem

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by dutyu** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/catalog/RefreshManager.java [hunk 1]
```diff
@@ -17,12 +17,7 @@
 
 package org.apache.doris.catalog;
 
-import org.apache.doris.analysis.CreateTableStmt;
-import org.apache.doris.analysis.DropTableStmt;
-import org.apache.doris.analysis.RefreshCatalogStmt;
-import org.apache.doris.analysis.RefreshDbStmt;
-import org.apache.doris.analysis.RefreshTableStmt;
-import org.apache.doris.analysis.TableName;
+import org.apache.doris.analysis.*;
```
Comment (Jibing-Li) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/RefreshManager.java:20]: This * will fail the code format check

**Review by Jibing-Li — APPROVED** [review]
LGTM

**Review by Jibing-Li — APPROVED** [review]

**Comment by dutyu** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**